wandb/integration/openai/fine_tune.py

```diff
@@ -0,0 +1,352 @@
+import datetime
+import io
+import json
+import os
+import re
+import time
+
+import wandb
+from wandb.sdk.lib import telemetry
+
+try:
+    import openai
+    from openai import OpenAI
+    from pkg_resources import parse_version
+
+    openai_version = openai.__version__
+    if parse_version(openai_version) < parse_version("1.0.1"):
+        raise wandb.Error(
+            f"This integration requires openai version 1.0.1 and above. Your current version is {openai_version}"
+        )
+except ImportError as e:
+    raise Exception(
+        "Error: `openai` not installed >> This integration requires openai!  To fix, please `pip install openai`"
+    ) from e
+
+try:
+    import numpy as np
+except ImportError as e:
+    raise Exception(
+        "Error: `numpy` not installed >> This integration requires numpy!  To fix, please `pip install numpy`"
+    ) from e
+
+try:
+    import pandas as pd
+except ImportError as e:
+    raise Exception(
+        "Error: `pandas` not installed >> This integration requires pandas!  To fix, please `pip install pandas`"
+    ) from e
+
+
+class WandbLogger:
+    """Log OpenAI fine-tunes to [Weights & Biases](https://wandb.me/openai-docs)."""
+
+    _wandb_api = None
+    _logged_in = False
+    openai_client = None
+
+    @classmethod
+    def sync(
+        cls,
+        id=None,
+        openai_client=None,
+        n_fine_tunes=None,
+        project="OpenAI-Fine-Tune",
+        entity=None,
+        force=False,
+        blocking=True,
+        **kwargs_wandb_init,
+    ):
+        """Sync fine-tunes to Weights & Biases.
+
+        :param id: The id of the fine-tune (optional)
+        :param openai_client: Pass the `OpenAI()` client (optional)
+        :param n_fine_tunes: Number of most recent fine-tunes to log when an id is not provided. By default, every fine-tune is synced.
+        :param project: Name of the project where you're sending runs. By default, it is "GPT-3".
+        :param entity: Username or team name where you're sending runs. By default, your default entity is used, which is usually your username.
+        :param force: Forces logging and overwrite existing wandb run of the same fine-tune.
+        :param blocking: Waits for the fine-tune to be complete and then log metrics to W&B. By default, it is True.
+        """
+        if openai_client is None:
+            openai_client = OpenAI(
+                api_key=os.environ["OPENAI_API_KEY"],
+            )
+        cls.openai_client = openai_client
+
+        if id:
+            print("Retrieving fine-tune job...")
```

Comment:
hmmm... do you want to use a generic print? you could use wandb printing logic (it will add the wandb prefix)

Reply:
I would like to use the wandb printing logic. Can you let me know the usage?